src/components/EditorsColumn.jsx

```diff
@@ -13,58 +17,99 @@ function allErrorsFor(language, errors, runtimeErrors) {
   return errors[language].items;
 }
 
-export default function EditorsColumn({
-  currentProject,
-  errors,
-  onComponentMinimize,
-  onEditorInput,
-  onRequestedLineFocused,
-  runtimeErrors,
-  ui,
-}) {
-  const editors = [];
-  ['html', 'css', 'javascript'].forEach((language) => {
-    if (includes(ui.minimizedComponents, `editor.${language}`)) {
-      return;
-    }
+function extractNodeHeights(editorRefs) {
+  return editorRefs.map(ref =>
+    ref ? ReactDOM.findDOMNode(ref).offsetHeight : null);
+}
+
+export default class EditorsColumn extends React.Component {
+  constructor(props) {
+    super(props);
+    bindAll(this, '_storeEditorRef', '_handleEditorDividerDrag');
+  }
+
+  _storeEditorRef(index, editor) {
+    this.editorRefs[index] = editor;
+  }
+
+  _handleEditorDividerDrag(index, _, {deltaY, lastY, y}) {
+    const {onUpdateFlex} = this.props;
+    const heights = extractNodeHeights(this.editorRefs);
+    onUpdateFlex({deltaY, heights, index, lastY, y});
+  }
 
-    editors.push(
-      <EditorContainer
-        key={language}
-        language={language}
-        source={currentProject.sources[language]}
-        onMinimize={
-          partial(onComponentMinimize, `editor.${language}`)
-        }
-      >
-        <Editor
-          errors={allErrorsFor(language, errors, runtimeErrors)}
+  render() {
+    const {
+      currentProject,
+      editorsFlex,
+      errors,
+      onComponentMinimize,
+      onEditorInput,
+      onRequestedLineFocused,
+      runtimeErrors,
+      ui,
+    } = this.props;
+
+    const editors = [];
```

Comment:
Very nitpicky but maybe we should call this variable something like `children` since it now contains things besides editors : )